doc/sphinx/addendum/omega.rst

```diff
@@ -7,12 +7,12 @@ Omega: a solver for quantifier-free problems in Presburger Arithmetic
 
 .. warning::
 
-   The :tacn:`omega` tactic is about to be deprecated in favor of the
-   :tacn:`lia` tactic.  The goal is to consolidate the arithmetic
-   solving capabilities of Coq into a single engine; moreover,
-   :tacn:`lia` is in general more powerful than :tacn:`omega` (it is a
-   complete Presburger arithmetic solver while :tacn:`omega` was known
-   to be incomplete).
+   The :tacn:`omega` tactic is deprecated in favor of the :tacn:`lia`
+   tactic.  The goal is to consolidate the arithmetic solving
+   capabilities of Coq into a single engine; moreover, :tacn:`lia` is
+   in general more powerful than :tacn:`omega` (it is a complete
+   Presburger arithmetic solver while :tacn:`omega` was known to be
+   incomplete).
 
    Work is in progress to make sure that there are no regressions
```

Comment:
Sorry @vbgl for not seeing this earlier, but this paragraph should be reformulated as well. Now, we recommend switching all existing code to `lia` and reporting any issue. If no new issues are reported, `omega` will be removed soon.

Reply:
Done.